core/sys/darwin/Foundation/NSString.odin

```diff
@@ -5,7 +5,7 @@ foreign import "system:Foundation.framework"
 @(objc_class="NSString")
 String :: struct {using _: Copying(String)}
 
-StringEncoding :: enum UInteger {
+StringEncoding :: enum u32 {
```

Comment:
This is `UInteger` in my headers, which is an `unsigned long` `c.ulong`.

Reply:
Nice catch! I got confused once I moved this type to CoreFoundation and forgot to revert this change as well.